CMakeLists.txt

```diff
@@ -219,7 +219,12 @@ else()
   set(CYTHON_EXECUTABLE "")
 endif()
 
-find_program(SYSTEMCTL_BIN systemctl HINTS "/usr/bin" "/bin")
+find_program(SYSTEMCTL_BIN_DEFAULT systemctl HINTS "/usr/bin" "/bin")
```

Comment:
I'm missing something important.

The issue is that debian doesn't have systemctl, but this code makes cmake to fail if systemctl doesn't exist. So what is this commit fixing?

Reply:
As discussed in the PR comments, the goal of this is to not be able to built a broken package.
Either systemctl is present and we autodetect it, or the distro/build can specify the path to it.

Reply:
We still support centos6, so breaking the build without systemd doesn't seem like an option? Confused by this remark

Reply:
My bad. I thought we only had systemd based distro left.